os/pm/pm_update.c

```diff
@@ -242,8 +237,8 @@ void pm_update(int domain, int16_t accum)
 		if (Y > g_pmexitthresh[index]) {
 			/* Yes... reset the count and recommend the normal state. */
```

Comment:
is there any use of these comparison of exit and enter thresholds ? becuase as we remove pm_activity() , i dont think accum variable is changed by any other api, so accum value is always 0 now.  So state change is happening either if we do pm_changestate() or if time interval increases from last state threshold count.
So can we remove unecessary comparision of accum value with enter and exit thresholds? @edwakuwaku @ewoodev @gSahitya-samsung